yarn.lock

```diff
@@ -16185,7 +16185,12 @@ semver-truncate@^3.0.0:
   resolved "https://registry.yarnpkg.com/semver/-/semver-5.7.2.tgz#48d55db737c3287cd4835e17fa13feace1c41ef8"
   integrity sha512-cBznnQ9KjJqU67B52RMC65CMarK2600WFnbkcaiwWq3xy/5haFJlshgnpjovMVJ+Hff49d8GEn0b87C5pDQ10g==
 
-semver@6.3.0, semver@6.3.1, semver@^6.0.0, semver@^6.1.1, semver@^6.1.2, semver@^6.3.0, semver@^6.3.1:
+semver@6.3.0:
+  version "6.3.0"
+  resolved "https://registry.yarnpkg.com/semver/-/semver-6.3.0.tgz#ee0a64c8af5e8ceea67687b133761e1becbd1d3d"
+  integrity sha512-b39TBaTSfV6yBrapU89p5fKekE2m/NwnDocOVruQFS1/veMgdzuPcnOM34M6CwxW8jH/lxEa5rBoDeUwu5HHTw==
+
+semver@^6.0.0, semver@^6.1.1, semver@^6.1.2, semver@^6.3.0, semver@^6.3.1:
```

Comment:
@jamesgorrie I'm not sure I follow this change 🤔. It looks like the net effect would be to downgrade the version of semver to 6.3.0, which was originally upgraded away from in #8715?

Reply:
Good catch @JamieB-gu yes `@pm2/io@~5.0.0` is still using semver@6.3.0. In which case we need to keep the resolution.

Reply:
Interestingly `pm2@5.3.0`'s dependency on `@pm2/io@~5.0.0` has been updated to address the vulnerability in `@pm2/io@5.0.2`:

https://github.com/keymetrics/pm2-io-apm/commit/c5996e3464b94fc95d3ff911160d51a599270403

As pm2 accepts a patch this should mean we can specifically install/upgrade `pm2/io@~5.0.2` to pick up this update?

Reply:
I think there has been a few issues where `yarn` was following `semver` so not updating it's lockfile as it was technically backwards compatible.

So basically the same thing you're talking about @arelra.

I removed `pm2` and re-added it which seems to have given us a more expected lockfile, but annoyingly we do still have high vulnerabilities, which I think are related to licensing.